sctptransport.go

```diff
@@ -369,3 +366,13 @@ func (r *SCTPTransport) generateAndSetDataChannelID(dtlsRole DTLSRole, idOut **u
 
 	return &rtcerr.OperationError{Err: ErrMaxDataChannelID}
 }
+
+func (r *SCTPTransport) Association() *sctp.Association {
```

Comment:
I don't believe we should export this. It isn't part of the ORTC ond webrtc-pc API

Reply:
👌
renamed to unexported `association()`